Filters/exclusions.txt

```diff
@@ -826,7 +826,8 @@ googletagmanager.com
 omtrdc.net
 scorecardresearch.com
 ! https://github.com/AdguardTeam/AdguardDNS/issues/89
-fwmrm.net
+/^.fwmrm.net\^/
```

Comment:
damn, merged. But I think it is not a problem here)

Reply:
Okay then.

Reply:
I just meant that current regexp will also match something like these:
```adblock
afwmrm.net^
afwmrmanet^
```
because dot is not escaped so it match any character.

Reply:
@AdamWr https://github.com/AdguardTeam/AdGuardSDNSFilter/pull/1386